pkg/scyllaclient/client_scylla.go

```diff
@@ -862,7 +866,7 @@ func (c *Client) TableDiskSizeReport(ctx context.Context, hostKeyspaceTables Hos
 
 		size, err := c.TableDiskSize(ctx, v.Host, v.Keyspace, v.Table)
 		if err != nil {
-			return parallel.Abort(errors.Wrapf(err, v.Host))
+			return parallel.Abort(errors.Wrapf(stdErrors.Join(err, ErrHostInvalidResponse), v.Host))
```

Comment:
So this error should be included in every client function, but right now we include it only in 1 repair specific call?
This sounds like it could create confusion in the future.

Reply:
No, it's just an option. You can include it, if you want to have a possibility to check one of the root causes.